src/react/theme/ThemeHelper.js

```diff
@@ -10,7 +10,8 @@ export const getHeaderTheme = (theme) => {
         getBgBorderBottomColor: () => getColorEffect(theme.color.accent, 20),
         getTextColor: () => theme.color.header?.text || headerTextColor,
         getToolColor: () => theme.color.header?.icon || funcs.getTextColor(),
-        getToolHoverColor: () => theme.color.accent
+        getToolHoverColor: () => theme.color.accent,
+        getOtherStyles: () => theme.otherStyles
```

Comment:
We can remove getOtherStyles() if we pass the theme through showPopup() options and add another helper for the font. Something like this:

```
export const getFont = (theme) => {
    if (theme.font === 'arial') return 'Arial, Helvetica, sans-serif';
    if (theme.font === 'georgia') return 'Georgia, Times, "Times New Roman"';
    return theme.font ||  'Arial, Helvetica, sans-serif';
}
```